Client/Frontend/Browser/Toolbars/BottomToolbar/Menu/Bookmarks/BookmarkDetailsView.swift

```diff
@@ -69,7 +72,10 @@ class BookmarkDetailsView: AddEditHeaderView, BookmarkFormFieldsProtocol {
         [emptySpacer, faviconImageView, textFieldsStackView]
             .forEach(contentStackView.addArrangedSubview)
         
-        if let url = url, let favUrl = URL(string: url) {
+        var url = url
+        if url?.isBookmarklet ?? false {
```

Comment:
I think in such unwrapped `if` conditions writing `url?.isBookmarklet == true` is easier to read